python/py-cartopy/Portfile

```diff
@@ -4,13 +4,14 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        SciTools cartopy 0.17.0 v
+github.setup        SciTools cartopy 0.18.0 v
 
 name                py-${github.project}
 revision            0
 categories-append   science gis graphics
 platforms           darwin
 license             LGPL-3
+version             0.18.0
```

Comment:
`github.setup` defines `version`, so there is no redundantly add the `version` field again.